### ISSUE TITLE
Relative asset import don't work until the CSS is changed

### ISSUE THREAD

**Issue by Den-dp** [angular/angular-cli#20919]
# 🐞 Bug report

### Command (mark with an `x`)
- [ ] new
- [ ] build
- [x] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] extract-i18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc

### Is this a regression?
Yes, the previous version in which this bug was not present was: 12.0.0

### Description
Image imported in CSS is not present in WDS after the first compilation.

## 🔬 Minimal Reproduction
1. run the following
```
git clone https://github.com/Den-dp/ng-relative-asset-import-example
cd ng-relative-asset-import-example
npm i
npm start
```
2. open http://localhost:4200 

**expected:**
served site should display `download.svg` as a background.

**actual:**
404 in the network and http://localhost:4200/webpack-dev-server confirms that there is no `download.svg` served by WDS.

Any change in `src/app/app.component.css` fixes the problem.

## 🔥 Exception or Error

<pre><code>
GET http://localhost:4200/download.svg 404 (Not Found)
</code></pre>

## 🌍 Your Environment

<pre><code>
Angular CLI: 12.0.1
Node: 14.15.5
Package Manager: npm 6.14.13
OS: win32 x64

Angular: 12.0.1
... animations, cli, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1200.1
@angular-devkit/build-angular   12.0.1
@angular-devkit/core            12.0.1
@angular-devkit/schematics      12.0.1
@schematics/angular             12.0.1
rxjs                            6.6.7
typescript                      4.2.4
</code></pre>

**Anything else relevant?**

Might be related to #20882 and #20885

**Comment by rudivanbiljon** [comment]
Same problem here. We have a complex hybrid setup and updating from 12 to 12.0.1 broke our css background-images from loading. The css itself is unchanged, but the files/locations have been changed.

**Comment by alan-agius4** [comment]
Duplicate of #20916 and #20882

**Comment by Den-dp** [comment]
@alan-agius4 I hope this will not be forgotten since both tickets are closed 🙂 🙏

**Comment by alan-agius4** [comment]
> @alan-agius4 I hope this will not be forgotten since both tickets are closed 🙂 🙏

That’s because it has already been fixed, and the fix will be available in 12.0.2
